### PR TITLE
TF-777: Add basic hover example

### PR DESCRIPTION
**Pull request by JordanBerni-UL**
- [x] Code Review
- [x] Developer Testing: Key item tests

### CODE REVIEW

File: Examples-Web/HoverButton/BasicHoverButton.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    new TouchFree.Cursors.SVGCursor();
+
+    // Add eventListers to all the buttons
+    const buttons = document.getElementsByClassName("button");
+    for (let i = 0; i < buttons.length; i++) {
+        const element = buttons[i];
+        // When the cursor enters the button then call onHover
+        element.addEventListener("pointerenter", () => onHover(element));
+        // When the cursor leaves the button then call onHover
+        element.addEventListener("pointerleave", () => onDehover(element));
```
Comment (cartmouse) [inline Examples-Web/HoverButton/BasicHoverButton.js]: Suggestion to change "dehover" to "unhover"? Dehover just hits my eye weirdly

File: Examples-Web/HoverButton/Version.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+TouchFree Tooling for Web Version: 1.1.1
+TouchFree Tooling API Version: 1.2.0
+Release Reference: deb6ae91
```
Comment (cartmouse) [inline Examples-Web/HoverButton/Version.txt:3]: Not seen this before in our demos, where's this getting read?

Comment (JordanBerni-UL) [reply Examples-Web/HoverButton/Version.txt:3]: It's present in the other button's folder (https://github.com/ultraleap/TouchFree-Tooling-Examples/tree/develop/Examples-Web/AnimatedProgressButtons). I copied this Version.txt from the Tooling release folder

Comment (JordanBerni-UL) [reply Examples-Web/HoverButton/Version.txt:3]: It isn't read anywhere - more so you know the version of the build tooling file

Comment (cartmouse) [reply Examples-Web/HoverButton/Version.txt:3]: Ah yeah, missed that one. Seems like a good thing :+1:

File: Examples-Web/HoverButton/BasicHoverButton.js (hunk 1 above)
Comment (JordanBerni-UL) [reply Examples-Web/HoverButton/BasicHoverButton.js]: Updated

**Review by cartmouse — APPROVED** [review]
Looks good to me!

**Pull request merged**